controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -621,70 +629,62 @@ def _prepare_mocks_for_get_host_by_identifiers_(self, result_reader_iter, hosts)
 
     @patch("controller.array_action.svc_cli_result_reader.SVCListResultsReader.__iter__")
     def test_get_host_by_identifiers_returns_host_not_found(self, result_reader_iter):
-        host_1 = self._get_host_as_dictionary('host_id_1', 'test_host_1', ['iqn.test.1'], [])
-        host_2 = self._get_host_as_dictionary('host_id_2', 'test_host_2', ['iqn.test.2'], [])
-        host_3 = self._get_host_as_dictionary('host_id_3', 'test_host_3', ['iqn.test.3'], [])
-        hosts = [host_1, host_2, host_3]
-        self._prepare_mocks_for_get_host_by_identifiers_(result_reader_iter, hosts)
+        self._prepare_mocks_for_get_host_by_identifiers_(result_reader_iter)
         with self.assertRaises(array_errors.HostNotFoundError):
-            self.svc.get_host_by_host_identifiers(Initiators('Test_iqn', ['Test_wwn']))
+            self.svc.get_host_by_host_identifiers(Initiators('Test_iqn', ['Test_WWN'], ''))
```

Comment:
I think it's usually:
```suggestion
            self.svc.get_host_by_host_identifiers(Initiators('Test_iqn', ['Test_wwn'], ''))
```
any reason not to keep it aligned?

Reply:
done